.ci/gitlab/.gitlab-ci.yml

```diff
@@ -175,6 +175,21 @@ default:
 .build:
   extends: [ ".base-job" ]
   stage: build
+  # Retry for everything but "legitimate build failures" (script_failure)
+  retry:
+    max: 2
```

Comment:
Move this under `.generate-common`, this job template is only used for the triggered jobs.